src/main/java/org/mybatis/typehandlers/threetenextra/AmPmTypeHandler.java

```diff
@@ -0,0 +1,66 @@
+/**
+ *    Copyright 2016-2017 the original author or authors.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.mybatis.typehandlers.threetenextra;
+
+import java.sql.CallableStatement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.apache.ibatis.type.BaseTypeHandler;
+import org.apache.ibatis.type.JdbcType;
+import org.threeten.extra.AmPm;
+
+/**
+ * Type Handler for threeten-extra AmPm
+ * 
+ * This type handler makes some assumptions on the data type of the column.
+ * If your RDBMS supports enums you are set.
+ * 
+ * @author Björn Raupach
+ */
+public class AmPmTypeHandler extends BaseTypeHandler<AmPm> {
```

Comment:
@harawata @raupachz

Is need this class ? I think we can use the `EnumTypeHandler` as alternative because The `AmPm` is enum class.
What do you think ?